No keys

eoliveros opened the pull request.

eoliveros committed:
Test file

eoliveros committed:
Deleted test file

eoliveros committed:
Modified the ansible_deploy.sh script to be able to run with only one parameter

djpnewton committed:
Merge branch 'master' of https://github.com/djpnewton/zapd into no_keys

djpnewton commented on ansible_deploy.sh:
```diff
@@ -25,70 +25,93 @@ display_usage() {
+
+#ADMIN_EMAIL=admin@zap.me
+#ALERT_EMAIL=alerts@zap.me
+ALERT_EMAIL=e.f.oliveros@gmail.com
```
careful about whats left in the commits

djpnewton commented on ansible_deploy.sh:
```diff
@@ -99,16 +122,16 @@ echo "   - BACKUP_HOST: $BACKUP_HOST"
-    ansible-playbook --inventory "$DEPLOY_HOST," --user "$DEPLOY_USER" -v \
-        --extra-vars "ADMIN_EMAIL=$ADMIN_EMAIL ALERT_EMAIL=$ALERT_EMAIL DEPLOY_HOST=$DEPLOY_HOST BACKUP_KEY='$BACKUP_KEY' BACKUP_SSH_KEY='$BACKUP_SSH_KEY' BACKUP_HOST=$BACKUP_HOST WEBHOOK_URL=$WEBHOOK_URL WEBHOOK_KEY=$WEBHOOK_KEY REMOTE_WAVES_NODES=$REMOTE_WAVES_NODES VAGRANT=$VAGRANT TESTNET=$TESTNET" \
+	## do dangerous stuff
+	echo ok lets go!!!
```
there is no need to change the whitespace

djpnewton committed:
-removed modidied whitespace
-made KEYS_SUPPLIED used similarly to TESTNET
-simplified use of KEYS_SUPPLIED
-removed test variables accidentially left in place

djpnewton committed:
-add option to only do minimal zapd deploy
-fix webhook config data being clobbered when doing a no keys deploy
